README.mkdn

```diff
@@ -0,0 +1,29 @@
+YAAP
+==============
+
+Getting Started
+---------------
+
+To get started with Android, you'll need to get
+familiar with [Git and Repo](http://source.android.com/source/using-repo.html).
+
+To initialize your local repository using the YAAP trees, use a command like this:
+
+```bash
+repo init -u git://github.com/yaap/manifest.git -b eleven
+```
+Then to sync up:
+```bash
+repo sync --force-sync --no-tags --no-clone-bundle
```

Comment:
not force sync would trigger ido, and add a nproc option.

Reply:
kek